src/result.rs

```diff
@@ -15,6 +15,8 @@
 #[allow(non_camel_case_types)]
 pub type z_result_t = i8;
 
+pub const Z_CHANNEL_DISCONNECTED: z_result_t = 1;
```

Comment:
I think these codes should be errors, i.e. value < 0

Reply:
These are pretty valid cases, for example reply_channel will return Z_CHANNEL_DISCONNECTED after you receive the last reply from get (there is no other way to get this information in case of channel, so I do not think that the fact of receiving it should be treated as a error). Also a constant similar to Z_CHANNEL_DISCONNECTED is defined as +1, in zenoh-pico.